### ISSUE TITLE
Active Player Count on map pages counts bots

### ISSUE THREAD

**Issue by Arhowk** [ModDota/BugTracker#82]
At the current moment the map "Legends of Dota: 7.0.0 Official" is being shown as a player count of 472 but it isn't on the top 10 list. Why?

Because most of those players are bots in lobbies like this one

https://getdotastats.com/#s2__match?id=19590065

1 player, 19 bots but it adds the player count to 20. This is being used to artificially inflate player counts and make games seem more popular than they are and confuse users about the ordering of the top 10 list.

TL;DR Bots shouldn't count for a game's active players.

**Comment by darklordabc** [comment]
Just saying, its not confirmed that this is because of bots.

**Comment by DoctorGester** [comment]
These 2 games share a foundation codebase so it's possible that the reason is not bots. My game had a similar issue due to having "dota_surrender_on_disconnect 0" command executed which inflated both player and spectator numbers towards the unreal amounts (over 100 spectators at some point).

**Comment by Arhowk** [comment]
I wasn't able to reproduce this using a simple test case and 9 instances of Tutorial:AddBot but this begs another question-

The custom game mode "Bomberman" also uses bots and also seems to not be inflated by this. However, both of these game modes (the one I made and Bomberman) do not have dedicated servers so it is impossible to confirm or deny this without either ripping apart a map with dedicated servers or getting a test bed.

There's some data to be gathered from Dota 2 redux's graphs ( https://getdotastats.com/#s2__mod?id=142&t=4 ) but I don't have the data on when the game did and didn't have dedi's (but there are obvious points of interest throughout) (Edit: October 22, 2016 is the date that Redux got dedicated servers which can point to a correlation; however around that time a known youtuber also posted a video about the mod which can contribute to some? increase in player count, however assuming the video had about 80,000 views at the time it's unrealistic to think that the video is responsible for the 900 additional players at the time) (Edit 2: The video still only has about 50,000 views right now so it can be estimated to have about 10,000 views at the time of incidence, 900 players means almost 10% of all of his viewers went out and immediately played the game. Completely unrealistic to think this). (Edit 3: The game was DMCA'd right after this rise in player count which may account for the sudden plummet of players and the slow rise back up)

If anyone has examples of games that either use bots (with or without dedis) or show this same level of artificially inflated player count (or artificially deflated ranking) with or without bots post it here

> These 2 games share a foundation codebase 

I never mentioned Redux before this comment (Edit: to clarify, we were discussing both Redux and LoD in the discord about this phenomenon but the original post only discusses LoD)

**Comment by Arhowk** [comment]
Not important
